conf/local.conf.sample

```diff
@@ -243,9 +243,6 @@ RDEPENDS_packagegroup-core-full-cmdline-sys-services_remove=" nfs-utils at tcp-w
 PACKAGE_EXCLUDE+=" ed"
 RDEPENDS_packagegroup-core-full-cmdline-utils_remove = " ed"
 
-PACKAGE_EXCLUDE+=" wireless-tools"
-RDEPENDS_packagegroup-base-wifi_remove = " wireless-tools"
-
```

Comment:
These PACKAGE_EXCLUDE, RDEPENDS_packagegroup-xyz should be moved to the images bb files. RDEPENDS_packagegroup_xyz lines remain the same, PACKAGE_EXCLUDE lines becomes IMAGE_INSTALL_remove.